jni/rec/rec_shim.c

```diff
@@ -0,0 +1,35 @@
+#include <dlfcn.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "rec_shim.h"
+
+#ifdef DEBUG
+#define DEBUG_PRINT(fmt, args...) fprintf(stderr, "msp_osd.rec_shim: " fmt "\n", ##args)
+#else
+#define DEBUG_PRINT(fmt, args...)
+#endif
+
+duss_osal_priority_t (*duss_osal_task_create_orig)(
+    duss_osal_task_attrib_t *task_attrib,
+    duss_osal_task_handle_t **task_handle) = NULL;
+
+extern gs_lv_transcode_t *rec_lv_transcode;
+
+duss_osal_priority_t duss_osal_task_create(
+    duss_osal_task_attrib_t *task_attrib,
+    duss_osal_task_handle_t **task_handle)
+{
+  if (duss_osal_task_create_orig == NULL)
+  {
+    duss_osal_task_create_orig = dlsym(RTLD_NEXT, "duss_osal_task_create");
```

Comment:
Has this been reliable for you? I've had issues with RTLD_NEXT returning NULL and having to load the lib manually for some libs. If this is reliable though it does save a lot of jankiness

Reply:
No problems here, at least with this function!